### PR TITLE
Added support to custom flags in install task.

### PR DESCRIPTION
**Pull request by arok**
Flag ‘-i’ is useful when test in-app billing.This PR added support for this custom flags.

### CODE REVIEW

**Comment by hal9002** [comment]
Can one of the admins review this PR?

**Comment by stefanhoth** [comment]
OK to test.

**Comment by stefanhoth** [comment]
@arok Could you please add a bit more info about this flag?

**Comment by arok** [comment]
This flag is emulate install package from market, like a Google Play or Yandex.Store.
When you try to call [PackageManager.getInstallerPackageName](http://developer.android.com/reference/android/content/pm/PackageManager.html#getInstallerPackageName(java.lang.String)) it's return package name of store, like for  application installed from market.
It's useful when debug in-app billing feature. You don't need upload every build of your application in market.

File: sample/build.gradle [hunk 1]
```diff
@@ -109,6 +109,12 @@ android {
                 println it
             }
         }
+
+        task('installFromGooglePlay', com.novoda.gradle.command.Install, ['assemble']) {
```
Comment (friedger) [inline sample/build.gradle]: Could you remove assemble here to make it consistent with the original install command? Or rename to assembleAndInstallFromGooglePlay

Comment (arok) [inline sample/build.gradle]: Removed

**Comment by friedger** [comment]
Thank you for your contribution!

**Comment by xrigau** [comment]
*Today I Learned*

**Pull request merged**